### ISSUE TITLE
First connection Defaul email and Password?

### ISSUE THREAD

**Issue by mariushosting** [assetgrid/assetgridapp#35]
What is the default email and password to log in for the first time?

**Comment by alex6480** [comment]
There is no default login. If a user doesn't exist, the login page redirect to a signup page, that currently looks almost exactly like the login page. So I don't blame you if you didn't notice that happened.

You first enter your desired login on the signup page, which will then redirect to the login page where you must enter it again to actually login. After the first user has been created, the signup page is disabled.

If you accidentally created a user and don't remember the credentials, you will need to reset the database.

**Comment by mariushosting** [comment]
The issue, is that. It's not working. Even if I add all the credentials, the page is frozen. Nothing happens.

![notwork](https://user-images.githubusercontent.com/37554361/195690639-6adfc2e4-ef78-44c1-9122-b5bfc7be25e5.png)

**Comment by mariushosting** [comment]
Ok I found the issue. The container does not work with mariaDB. Instead it works with SQLite.

Here is my docker compose that does not work and does not redirect to the signup page, only sign in page.

```
version: "3.1"
services:
  db:
    image: mariadb:latest
    container_name: Assetgrid-DB
    restart: always
    volumes:
      - /volume1/docker/assetgridatabase:/var/lib/mysql
    environment:
      MYSQL_ROOT_PASSWORD: secret
      MYSQL_PASSWORD: secret
      MYSQL_DATABASE: assetgrid
      MYSQL_USER: assetgrid
  assetgrid:
    image: assetgrid/assetgrid
    container_name: Assetgrid
    restart: always
    links:
      - db:db
    environment:
      Provider: "MySQL"
      "ConnectionStrings:MySQL": "Server=db;Database=assetgrid;Uid=assetgrid;Pwd=secret"
    ports:
      - 5160:8080
```

**Comment by alex6480** [comment]
That's weird. Could you go into the network tab on your browser devtools and see if the request is failing?
If you don't know how, I can guide you.

Here's an example of how that tab looks on Firefox
![billede](https://user-images.githubusercontent.com/8026279/195695524-beb19aab-52c8-4d6e-959a-9130924feb6a.png)

**Comment by mariushosting** [comment]
Please ready my previous message.

**Comment by alex6480** [comment]
Is there anything in the docker log? Maybe the backend program has trouble starting?

**Comment by mariushosting** [comment]
> Is there anything in the docker log? Maybe the backend program has trouble starting?

No errors, everything has started correctly. I will use SQLite from now on. Maybe you can take a look at why it's not working with mariaDB.

**Comment by alex6480** [comment]
The issue is that I am using mariaDB myself, which runs without errors. But I'll try your compose file, maybe that will reproduce the issue.

**Comment by verchalent** [comment]
I used the stock compose file from the website and ran into several issues. Aside from needing to add lines to create the DB and set creds, the primary container was crashing. Once I got to a compose file similar to @mariushosting everything came up fine, but I am now hitting the same issue. No matter what I do, I am redirected to /login. What is the page it should be redirecting to (signup push me right back to login)? 

@mariushosting what does your working docker-compose look like?

**Comment by alex6480** [comment]
There was an issue during the initial database migration, which has been fixed in 0.2.0

Assetgrid should now start without issue with MySQL

**Comment by decisoft** [comment]
Hi again, sorry to reopen this issue, but it doesn't start without issue if I want MySQL.

I copied the docker-compose example just as it is on the README file:
`version: "3.1"
services:
  db:
    image: mariadb:latest
    container_name: Assetgrid-DB
    restart: always
    environment:
      MYSQL_ROOT_PASSWORD: secret
      MYSQL_PASSWORD: secret
      MYSQL_DATABASE: assetgrid
      MYSQL_USER: assetgrid

  assetgrid:
    image: assetgrid/assetgrid
    container_name: assetgrid
    links:
      - db:db
    environment:
      Provider: "MySQL"
      "ConnectionStrings:MySQL": "Server=db;Database=assetgrid;Uid=assetgrid;Pwd=secret"
    ports:
      - 8595:8080

networks: 
  default:
    external: true
    name: lan
`

I only modified the network and the port, as the 80 was used by Traefik. It doesn't start the app container: 
`Unhandled exception. MySqlConnector.MySqlException (0x80004005): Unable to connect to any of the specified MySQL hosts.`

What am I missing?

Thanks
